src/manifest/MsQuicEtw.man

```diff
@@ -3012,905 +10059,3493 @@
+            id="CLOG.PeerPreferredAddress"
+            value="[conn][%0 ] Peer configured preferred address %1 "
+            />
+        <string
+            id="CLOG.NegotiatedDisable1RttEncryption"
+            value="[conn][%0 ] Negotiated Disable 1-RTT Encryption"
+            />
+        <string
+            id="CLOG.CustomCertValidationPending"
+            value="[conn][%0 ] Custom cert validation is pending"
+            />
+        <string
+            id="CLOG.RecvStatelessReset"
+            value="[conn][%0 ] Received stateless reset"
+            />
+        <string
+            id="CLOG.HandshakeConfirmedFrame"
+            value="[conn][%0 ] Handshake confirmed (frame)"
+            />
+        <string
+            id="CLOG.FirstCidUsage"
+            value="[conn][%0 ] First usage of SrcCid: %1 "
+            />
+        <string
+            id="CLOG.PathDiscarded"
+            value="[conn][%0 ] Removing invalid path[%1 ]"
+            />
+        <string
+            id="CLOG.Unreachable"
+            value="[conn][%0 ] Received unreachable event"
+            />
+        <string
+            id="CLOG.UpdateShareBinding"
+            value="[conn][%0 ] Updated ShareBinding = %1 "
+            />
+        <string
+            id="CLOG.UpdateStreamSchedulingScheme"
+            value="[conn][%0 ] Updated Stream Scheduling Scheme = %1 "
+            />
+        <string
+            id="CLOG.ApplySettings"
+            value="[conn][%0 ] Applying new settings"
+            />
+        <string
+            id="CLOG.RttUpdatedMsg"
+            value="[conn][%0 ] Updated Rtt=%1 .%2  ms, Var=%3 .%4 "
+            />
+        <string
+            id="CLOG.NewSrcCidNameCollision"
+            value="[conn][%0 ] CID collision, trying again"
+            />
+        <string
+            id="CLOG.ZeroLengthCidRetire"
+            value="[conn][%0 ] Can&apos;t retire current CID because it&apos;s zero length"
+            />
+        <string
+            id="CLOG.TimerExpired"
+            value="[conn][%0 ] %1  timer expired"
+            />
+        <string
+            id="CLOG.IndicateShutdownByPeer"
+            value="[conn][%0 ] Indicating QUIC_CONNECTION_EVENT_SHUTDOWN_INITIATED_BY_PEER [0x%1 ]"
+            />
+        <string
+            id="CLOG.IndicateShutdownByTransport"
+            value="[conn][%0 ] Indicating QUIC_CONNECTION_EVENT_SHUTDOWN_INITIATED_BY_TRANSPORT [0x%1 ]"
+            />
+        <string
+            id="CLOG.IndicateConnectionShutdownComplete"
+            value="[conn][%0 ] Indicating QUIC_CONNECTION_EVENT_SHUTDOWN_COMPLETE"
+            />
+        <string
+            id="CLOG.IndicateResumptionTicketReceived"
+            value="[conn][%0 ] Indicating QUIC_CONNECTION_EVENT_RESUMPTION_TICKET_RECEIVED"
+            />
+        <string
+            id="CLOG.ClientVersionNegotiationCompatibleVersionUpgrade"
+            value="[conn][%0 ] Compatible version upgrade! Old: 0x%1 , New: 0x%2 "
```

Comment:
I would fix this if I knew how to in CLOG, but it seems that CLOG is adding extra spaces after all the `%N`s in the strings.